### ISSUE TITLE
Unable to run 

### ISSUE THREAD

**Issue by aneesv** [gratimax/gulp-grunt#23]
Why its given `gruntCliDir + ` to spawn at 
`spawn(
        gruntCliDir + gruntCmd,
        args,
        {cwd: cwd}
);`

On providing a base its trying to execute <base>/grunt which gives error , on removing  `gruntCliDir + ` its working fine

**Comment by cellistlondon** [comment]
Agreed, the base is set elsewhere (`grunt.file.setBase(cwd);`) so there doesn't seem to be a reason to prepend the path here?

**Comment by noelmace** [comment]
:+1: 

Error: spawn C:\Users\blabla/grunt.cmd ENOENT
